index.html

```diff
@@ -14,6 +16,77 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <header>
+    <nav>
+      <a href="">Portfolio</a>
+      <a href="https://adventurini.github.io/portfolio-website/">Dev Projects</a>
+      <a href="">Blog</a>
+      <a href="">Contact</a>
+    </nav>
+
+  </header>
+  <div class="container">
+    <div class="heading">
+  <h1>ANTHONY VENTURINI
+  <div class="border">
+
+  </div>
+</h1>
+  <h2>FULL STACK DEVELOPER</h2>
+  <div class="table">
+  <ul id="horizontal-list"><li>HTML</li> <li>CSS</li> <li>JAVASCRIPT</li> <li>REACT</li>  <li>DJANGO</li> </ul>
+</div>
+<div class="border-2">
+
+  </div>
+
+</div>
+<div class="intro">
+<h3>INTRODUCTION</h3>
+<br>
+<p>With years of experience in real estate investing and sales,
+  coupled with analytical skills from playing poker on a world class,
+  professional level, I have quite a different vantage point and skillsets
+  than the ordinary.
+</p>
+<br>
+<br>
```

Comment:
I think the preferred way to create space between sections is margin rather than <br>'s.